### ISSUE TITLE
card calc little fix needed)

### ISSUE THREAD

**Issue by Diolik** [Excrulon/Tree-of-Savior-Lua-Mods#126]
i'm lvl 34... lvl card 10 used on lvl 100 )
![e4e4c64d54fd191fb9cd2f232fe9edda](https://cloud.githubusercontent.com/assets/19318760/15270415/dd93ebae-1a38-11e6-8cca-87cef5484375.png)

**Comment by TehSeph** [comment]
Haha. Yeah not including cards you can't use might be a good idea.

**Comment by Excrulon** [comment]
Yeah, I know about that but I don't think I'm going to fix it. I can easily do a level check but I don't feel like writing code to run it over and over again until you can't gain levels anymore. I just put those cards in my storage. Might get around to it some day...
